test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1

```diff
@@ -221,7 +223,7 @@ function RunSaveHelpTests
                 $saveHelpFolder = Join-Path $TestDrive (Get-Random).ToString()
                 New-Item  $saveHelpFolder -Force -ItemType Directory
 
-                It "Validate Save-Help for the '$moduleName' module" {
+                It "Validate Save-Help for the '$moduleName' module" -Pending:$Pending {
```

Comment:
You're introducing a new pattern here passing in the $Pending value.  My preference is to just have -Pending on the relevant test cases rather than trying to pass it by the switch.  One reason is I can search all the tests to see which ones are pending, but with this, I have to look at the code to determine if it's pending or not.